app/javascript/services/repos-service.js

```diff
@@ -0,0 +1,15 @@
+import axios from '../helpers/axios-rails';
+import {githubReposRoute, githubRepoRoute} from "./service-routes";
+
+class ReposService {
```

Comment:
If we have a service, why is the component using a Rails.ajax call instead? https://github.com/project-anacapa/anacapa-github-linker/pull/276/files#diff-eefadcfa595c58f570ad562d60059f85R57-R72